### ISSUE TITLE
numeric size with scale and precision

### ISSUE THREAD

**Issue by stanislas-m** [gobuffalo/fizz#8]
Moved from https://github.com/gobuffalo/pop/issues/114, cc @u007 

hi,

what is the proper way to add in scale and precision for numeric data type in fizz?
is this supported?

**Comment by markbates** [comment]
I would’ve thought you can just add them to the options, but maybe not. They might need to be added as supported options. 

/cc @u007

**Comment by u007** [comment]
yes previously i thought option size: "12,2" will work, but it seems not.
turn out doing this work, but i think it might causes problem if the database use another data type name.
```
add_column("table", "field1", "decimal(12,2)", {"default": 0.00})
```
